docs/install_guides/installing-openvino-conda.md

```diff
@@ -44,14 +44,35 @@ Installing OpenVINO Runtime with Anaconda Package Manager
 
       conda update --all
 
-3. Install the OpenVINO Runtime package:
+3. Install the OpenVINO™ Runtime package:
 
    .. code-block:: sh
 
       conda install -c conda-forge openvino=2023.0.0
 
    Congratulations! You have finished installing OpenVINO Runtime.
 
+Compiling with OpenVINO™ Runtime from Conda-Forge
+###########################################################
+
+When linking OpenVINO™ libraries from Conda, ensure you have the necessary Conda compilers installed. 
+To do so, run the following command in your Conda environment:
+
+.. code-block:: sh
+
+    conda install cmake c-compiler cxx-compiler make
```

Comment:
it's turned out the compilers are not required on macOS (at least 13.x). But maybe it depends on the macOS version and on 11.x with older libc++ we still require conda compilers.

Could you please try macOS 11? and Windows ?

Reply:
It's working on Windows and MacOS without installing compilers from Conda

Reply:
I've updated title